Wikipedia/Code/StorageAndSyncingSettingsViewController.swift

```diff
@@ -183,6 +182,7 @@ extension StorageAndSyncingSettingsViewController: UITableViewDelegate {
         guard let footer = self.tableView(tableView, viewForFooterInSection: section) as? WMFTableHeaderFooterLabelView else {
             return 0
         }
+        footer.prepareForReuse()
```

Comment:
Is there a reason you need to invoke this directly at all? I thought `prepareForReuse` was invoked by the framework automatically?

Or is there a side effect here of calling this? (maybe this is also a pattern that is being use elsewhere?)

Reply:
nm… out of date